assets/src/edit-story/elements/media/scalePanel.js

```diff
@@ -39,15 +41,15 @@ const Container = styled.div`
   top: ${({ y, height }) => `${y + height + OFFSET_Y}px`};
   width: ${({ width }) => `${Math.max(width, MIN_WIDTH)}px`};
   height: ${HEIGHT}px;
-
   background: ${({ theme }) => theme.DEPRECATED_THEME.colors.t.bg};
-  border-radius: 100px;
+  border-radius: 8px;
+  padding: 3px ${HORIZONTAL_PADDING}px;
+  margin-top: 8px;
+`;
 
-  display: flex;
-  flex-direction: row;
-  flex-wrap: nowrap;
-  align-items: center;
-  padding: 0 4px;
+const ScaleSlider = styled(Slider)`
+  width: ${({ width }) =>
+    Math.max(width, MIN_WIDTH) - 2 * HORIZONTAL_PADDING}px;
```

Comment:
For some reason, the thumb highlight is sometimes off-by-one:
![Screen Shot 2021-02-21 at 23 07 51](https://user-images.githubusercontent.com/637548/108661572-1e798c80-749a-11eb-998e-820117e91782.png)


Not sure how to fix that without making a completely custom slider. It could be done by styling the thumb with a radial gradient, but that's not gonna look very sharp, unfortunately. And there's no pseudo-pseudo-element (so no `::-webkit-slider-thumb::after`).

Reply:
@barklund -- reworked the focus state logic and now a  border is used for the thumb and the thumb margin is adjusted instead. PTAL. I suppose there's a possibility that the thumb shifts a bit when it gets to focus state (although didn't notice it when testing currently) but it definitely shifts less than before redesign and it's now in sync with the thumb.